multipart-form-writer/progressbar.ml

```diff
@@ -0,0 +1,39 @@
+let erase_progress_bar size =
```

Comment:
This files needs a `.mli`.  By the way, I think `progress_bar` would be a better name since "progress bar" is in two words.